hack/tidy-check.sh

```diff
@@ -26,8 +26,13 @@ SUM_FILE="$PROJECT_DIR/go.sum"
 TMP_DIR="$THIS_DIR/.tmp.tidy-check"
 TMP_MOD_FILE="$TMP_DIR/go.mod"
 TMP_SUM_FILE="$TMP_DIR/go.sum"
+OCTANT_PLUGIN_MOD_FILE="$PROJECT_DIR/plugins/octant/go.mod"
```

Comment:
I feel the script can be designed to check a project's go mod by specifying the directory.
Then we just need to call it with another directory when adding new sub-projects, instead of adding repeated code.

Reply:
Good point, thanks. I have updated this part by adding a new parameter instead of modifying tidy-check.sh directly. Please help take a look at the latest version.